trade-finance-manager-ui/server/helpers/premium-payments-table-checkbox-id-helper.ts

```diff
@@ -24,10 +42,14 @@ export const getFeeRecordPaymentCurrencyFromPremiumPaymentsCheckboxId = (checkbo
   return currency as Currency;
 };
 
+export const getFeeRecordIdsFromPremiumPaymentsCheckboxId = (checkboxId: PremiumPaymentsTableCheckboxId): number[] => {
```

Comment:
Can we add some documentation here please and for `getFeeRecordIdsFromPremiumPaymentsCheckboxIds`?


Finding it hard to digest very similar long function names, anyway we could shorten these?

```ts
getFeeRecordIdsFromPremiumPaymentsCheckboxIds
getFeeRecordIdsFromPremiumPaymentsCheckboxId
```

Reply:
There are other checkboxes within TFM UI (e.g. keying sheet) which also include fee record ids, so would like to keep `PremiumPayments` in the name of this.

We also have types that have the `CheckboxId` suffix in their name (e.g. `KeyingSheetCheckboxId` and `PremiumPaymentsTableCheckboxId`) so I think the current name is suitable even though it's a bit hard to distinguish the two

Reply:
Not sure what to suggest here, some of the functions here (old and new) are too long and hard to follow IMO

Maybe could be renamed to this, since they're in a file called "premium payments"

```ts
getFeeRecordIdsFromCheckboxId
getFeeRecordIdsFromMultipleCheckboxIds
```

But i'm not sure if this is wise without further context.

Reply:
I see your point that we might want to omit it due to the filename, I think it might be best to keep them with the premium payments reference though as this is keeping with how the keying sheets checkboxes are named